src/util/strencodings.cpp

```diff
@@ -569,3 +569,16 @@ std::string Capitalize(std::string str)
     str[0] = ToUpper(str.front());
     return str;
 }
+
+std::string HexStr(const Span<const uint8_t> s)
+{
+    std::string rv;
+    static const char hexmap[16] = { '0', '1', '2', '3', '4', '5', '6', '7',
```

Comment:
nit:
```suggestion
    static constexpr char hexmap[16] = { '0', '1', '2', '3', '4', '5', '6', '7',
```

Reply:
I'm not sure. Does this make a difference for an array?